Makefile

```diff
@@ -61,8 +61,7 @@ test: run-unit-tests
 
 .PHONY: lint
 lint: build-slim build-test
-	# Note: Make sure that you run `git config diff.noprefix false` in this repo
-	# See this issue for more details: https://github.com/golangci/golangci-lint/issues/948
+	if [ $$(git config --get diff.noprefix) == "true" ]; then echo -e "\nLinting won't work because you have a git setting 'diff.noprefix' set to 'true',\nwhich does not work reliably with the golangci-lint --new-from-rev flag. \n\nTo run the linter, set 'diff.noprefix' setting to 'true' for this repository using the following command:\n\n'git config diff.noprefix false'\n\nSee https://github.com/golangci/golangci-lint/issues/948 for more details."; exit 1; fi
```

Comment:
Why starting `\n` for the `echo`?

Reply:
To separate the output from make printing command to execute, to make it more readable.

Reply:
Oh, makes sense. Thanks!